main.js

```diff
@@ -0,0 +1,11 @@
+const allPosts = document.querySelector('li')
+allPosts.remove()
+
+const spanElmt = document.querySelector('span')
+spanElmt.remove()
+
+const stopPlan = document.querySelector('main .posts article')
+stopPlan.remove()
```

Comment:
I'm kinda surprised this passed the test. Since querySelector grabs the first article that matches the argument, your code is deleting the first instance of `main .posts article` instead of the final posts as mentioned in the instructions